libs/core/src/lib/popover/popover.component.scss

```diff
@@ -1,23 +1,125 @@
 @import '~fundamental-styles/dist/popover';
 
-//TODO to be reviewed and if needed to be moved in fd-styles
+
+$fd-popover-arrow-top-back: -0.5rem !default;
+$fd-popover-arrow-top-front: calc(-0.5rem + 1px) !default;
+
+$fd-popover-arrow-width: 1rem !default;
+$fd-popover-arrow-height: 0.5rem !default;
+$fd-popover-arrow-width-half: $fd-popover-arrow-width/2 !default;
+$fd-popover-arrow-offset: 0.5rem !default;
+$fd-popover-arrow-color: var(--sapContent_ForegroundBorderColor, #89919a) !default;
+$fd-popover-background-color: var(--sapGroup_ContentBackground, #fff) !default;
+
+.fd-popover__arrow {
```

Comment:
What are these styles? Something custom?

Reply:
This is styling from `fundamental-styles`, but with changed selectors. Current style's css requires to keep `x-placement` attribute along with `fd-popper` class. Also now arrow sticks to middle of trigger element, instead body. It should be added in styles in feature.